### ISSUE TITLE
icon for VTR is black-on-dark in the Firefox sidebar

### ISSUE THREAD

**Issue by sudoer** [Croydon/vertical-tabs-reloaded#293]
In Dark mode, in the Firefox sidebar, which contains icons for AI Chatbot, Tabs from other devices, History, Bookmarks, and Vertical Tabs Reloaded, the VTR icon is black on dark while the others are white-on-dark.

<img width="114" height="202" alt="Image" src="https://github.com/user-attachments/assets/203e9438-9a8f-4287-a9ce-aff3c3e8db6e" />

You almost can't see the VTR icon.

Mac OS 26, Firefox 145.0.2
